lib/html/pipeline/@mention_filter.rb

```diff
@@ -81,8 +81,8 @@ def call
         doc
       end
 
-      # The URL to provide when someone @mentions a "mention" name, such as
-      # @mention or @mentioned, that will give them more info on mentions.
+      # The URL to provide when someone @mentions a "mention" name, such
+      # [UnknownTag] Unknown tag @mentionas @mention or @mentioned, that will give them more info on mentions.
```

Comment:
Is there no way to do like `\@mention`? (I'm not familiar with YARD's details.)

Either way I think there's a typo as this has now been munged into `@mentionas`.

Reply:
Wow, what a typo! the error message pasted into the _Description_ is pasted verbatim in there.

All my edit was supposed to do was "reword in order to not have to `\@mention` which is the YARD escape sequence.

Reply:
Thanks for locating the issue, I added a commit with the typo fix.